scripts/ci-install-deps.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# Called from multiple CI pipelines in .github/workflows
+set -xeuo pipefail
+
+# native deps
+# FIXME: this is a lot and most aren't actually used in testing but tests will crash without due to GtkSource require
```

Comment:
This is slow but it does work.

With some further changes to avoid importing Gtk things during unit test runs this list could be a lot smaller, I'm not sure if that's worth doing.

Reply:
pushed a change to main, gtk dependencies are not required for unit tests anymore

Reply:
Nice, it's a lot faster now. I had to remove one more Gtk dependency in test_daemon.py which was unused.